src/pages/SpatialQuery/map.js

```diff
@@ -24,6 +24,7 @@ import 'ol-layerswitcher/dist/ol-layerswitcher.css';
 import '../../styles/App.css';
 import MapComponent from '../../components/MapComponent';
 import { ToggleDraw, RotateMap } from '../../components/MapControls';
+import Footer from '../../components/Footer';
 
 
 const SpatialMap = () => {
```

Comment:
@akshat22 / @nisarg210  the box drawn here doesn't rotate with the map. Check if it's an easy fix